### PR TITLE
Add probes to stats-api

### PR DESCRIPTION
**Pull request by pragyamehta**

### CODE REVIEW

**Comment by amsoedal** [comment]
LGTM!

**Review by amsoedal — APPROVED** [review]

File: samples/todo-app/deployment.yaml [hunk 1]
```diff
@@ -102,6 +102,18 @@ spec:
       containers:
       - name: stats-api
         image: azdspublic/todo-app-stats-api
+        livenessProbe:
```
Comment (lolodi) [inline samples/todo-app/deployment.yaml:105]: Should the other services also have probes?

**Review by lolodi — APPROVED** [review]

File: samples/todo-app/stats-api/server.js [hunk 2]
```diff
@@ -13,6 +13,11 @@ var cache = redis.createClient({
     password: process.env.REDIS_PASSWORD || undefined
 });
 
+app.get('/hello', function (req, res) {
```
Comment (daniv-msft) [inline samples/todo-app/stats-api/server.js:16]: Nit: should we use '/stats/hello', or something like that, as this is the probe for stats-api, and not for the whole app? That said, it's a sample app so it's not a big deal. Feel free to won't fix.

**Review by daniv-msft — APPROVED** [review]

File: samples/todo-app/deployment.yaml (hunk 1 above)
Comment (pragyamehta) [reply samples/todo-app/deployment.yaml:105]: They can, but it's not necessary. Probes are optional on kubernetes pods. The idea of this is to add probes to the service that is tested with bridge.

File: samples/todo-app/stats-api/server.js (hunk 2 above)
Comment (pragyamehta) [reply samples/todo-app/stats-api/server.js:16]: This endpoint is on the stats-api only. It is not available on any other microservice.

**Pull request merged**